solutions/task_2.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+"""
+TASK: Create 'Fake' class to run code below without errors.
+"""
+
+import random
+import string
+
+
+class Fake(object):
+    def __getattr__(self, name):
+        print("Non existing attribute {} has been called".format(name))
+        return self
+
+    def __call__(self, *args, **kwargs):
+        try:
+            super().__call__(self, *args, **kwargs)
+            return self
+        except AttributeError:
+            print("The attribute has been called with:")
+            if args:
+                print("- positional arguments: {}".format(args))
+            else:
+                print("- no positional arguments")
+            if kwargs:
+                print("- keyword arguments: {}".format(kwargs))
+            else:
+                print("- no keyword arguments")
+            return self
+
+    def __getitem__(self, item):
```

Comment:
Are you going to add ability to get existing item? (to be similar with call __call__)